client/devdocs/docs-example/wrapper.jsx

```diff
@@ -26,7 +26,6 @@ class DocsExampleWrapper extends Component {
 		name: PropTypes.string.isRequired,
 		unique: PropTypes.bool,
 		url: PropTypes.string.isRequired,
-		pageScroll: PropTypes.func.isRequired,
```

Comment:
Sorry, still need the `propType` definition, just not the `isRequired` :)

Reply:
Haha, of course!